pattoo_shared/installation/configure.py

```diff
@@ -0,0 +1,313 @@
+"""Methods for configuring pattoo components"""
+# Standard imports
+import os
+import grp
+import pwd
+import getpass
+
+# Dependendices
+import yaml
+
+# Import project libraries
+from pattoo_shared import files, configuration, log
+from pattoo_shared.installation import shared
+
+
+class Configure():
```

Comment:
We are going to have to review this. Many of your methods don't use class attributes and should therefore be functions.